tests/Aspire.Qdrant.Client.Tests/ConformanceTests.cs

```diff
@@ -10,20 +11,34 @@
 
 namespace Aspire.Qdrant.Client.Tests;
 
-public class ConformanceTests : ConformanceTests<QdrantClient, QdrantClientSettings>
+public class ConformanceTests : ConformanceTests<QdrantClient, QdrantClientSettings>, IClassFixture<QdrantContainerFixture>
 {
+    private readonly QdrantContainerFixture _containerFixture;
+
+    private readonly string _connectionString;
+
     protected override bool SupportsKeyedRegistrations => true;
 
     protected override bool CanCreateClientWithoutConnectingToServer => false;
 
-    protected override bool CanConnectToServer => AspireQdrantHelpers.CanConnectToServer;
+    protected override bool CanConnectToServer => RequiresDockerAttribute.IsSupported;
 
     protected override ServiceLifetime ServiceLifetime => ServiceLifetime.Singleton;
 
     protected override string[] RequiredLogCategories => Array.Empty<string>();
 
     protected override string ActivitySourceName => "";
 
+    protected override string? ConfigurationSectionName => "Aspire:Qdrant:Client";
+
+    public ConformanceTests(QdrantContainerFixture containerFixture)
+    {
+        _containerFixture = containerFixture;
+        _connectionString = RequiresDockerAttribute.IsSupported ?
+        $"{_containerFixture.GetConnectionString()}" :
+        "Endpoint=http://localhost:6334;Key=pass";
```

Comment:
```suggestion
            $"{_containerFixture.GetConnectionString()}" :
            "Endpoint=http://localhost:6334;Key=pass";
```

These should be intended from the parent line.